packages/flutter_tools/test/integration.shard/test_utils.dart

```diff
@@ -22,7 +22,8 @@ Directory createResolvedTempDirectorySync(String prefix) {
 void writeFile(String path, String content) {
   globals.fs.file(path)
     ..createSync(recursive: true)
-    ..writeAsStringSync(content);
+    ..writeAsStringSync(content)
+    ..setLastModifiedSync(DateTime.now().add(const Duration(seconds: 10)));
```

Comment:
avoid `DateTime.now()` in tests, it's a source of flakes. The memory file system has its own clock.

Reply:
These tests run on the flutter_tester and require the real filesystem. Unfortunately all of them are slightly flaky on Windows, and my current theory is that we don't have enough timestamp resolution there.